e2e/pkg/installation.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+//+build e2e
+
+package pkg
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/mattermost/mattermost-cloud/model"
+	"github.com/pkg/errors"
+)
+
+// WaitForInstallationAvailability pings installation until it responds successfully.
+func WaitForInstallationAvailability(dns string, log logrus.FieldLogger) error {
+	err := WaitForFunc(NewWaitConfig(20*time.Minute, 20*time.Second, 2, log), func() (bool, error) {
+		resp, err := http.Get(pingURL(dns))
+		if err != nil {
+			log.WithError(err).Error("Error while pining request")
```

Comment:
`pinging` is misspelled but the error message could also include a little more context for debugging:

```suggestion
			log.WithError(err).Errorf("Error while making ping request to Installation %s", dns)
```